mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -330,6 +371,52 @@ def _get_endpoint_from_method(self, method):
     def _get_all_endpoints_from_method(self, method):
         return _METHOD_TO_ALL_INFO[method]
 
+    def _call_endpoint(self, api, json_body, call_all_endpoints=False, extra_headers=None):
+        """
+        Override the base _call_endpoint to handle path parameter substitution for
+        Unity Catalog endpoints that use REST-style paths with {name}, {version}, etc.
+        """
+        response_proto = self._get_response_from_method(api)
+
+        if call_all_endpoints:
+            endpoints = self._get_all_endpoints_from_method(api)
+            return call_endpoints(
+                self.get_host_creds(), endpoints, json_body, response_proto, extra_headers
+            )
+        else:
+            endpoint, method = self._get_endpoint_from_method(api)
+
+            # Handle path parameter substitution for Unity Catalog prompt endpoints
+            if json_body and "{" in endpoint and "}" in endpoint:
+                # Parse the JSON body to extract path parameters
+                body_dict = json.loads(json_body) if isinstance(json_body, str) else json_body
+
+                # Extract all path parameters from the endpoint template
+                path_param_names = re.findall(r"\{(\w+)\}", endpoint)
```

Comment:
Same comment as https://github.com/mlflow/mlflow/pull/15946/files#r2114866044 - seems brittle, unless this is done elsewhere?